project/src/test/java/com/google/sps/EventTest.java

```diff
@@ -0,0 +1,80 @@
+package com.google.sps;
+import com.google.cloud.Date;
+import com.google.sps.data.Event;
+import com.google.sps.data.User;
+import com.google.sps.data.VolunteeringOpportunity;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+import org.junit.Assert;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
```

Comment:
Next time let's put test file and production code in the same PR.

Reply:
Ack-yes my bad!